### PR TITLE
Re-enable tests and fix compatibility with new photutils

### PR DESCRIPTION
**Pull request by mfisherlevine**

### CODE REVIEW

File: spectractor/extractor/targets.py [hunk 1]
```diff
@@ -212,28 +212,31 @@ def load(self):
...
-        # condition at import time, so putting here rather than at the
-        # module level so that multiple test runners don't run the race
-        from astroquery.simbad import Simbad
-        patchSimbadURL(Simbad)
-        Simbad.add_votable_fields('flux(U)', 'flux(B)', 'flux(V)', 'flux(R)', 'flux(I)', 'flux(J)', 'sptype',
-                                  'parallax', 'pm', 'z_value')
+        # explicitly make a class instance here because:
+        # when using from astroquery.simbad import Simbad and then using
+        # Simbad... methods secretly makes an instance, which stays around,
+        # has a connection go stale, and then raises an exception seeminly
```
Comment (erykoff) [inline spectractor/extractor/targets.py:218]: seemingly

File: spectractor/extractor/targets.py [hunk 2]
```diff
@@ -212,28 +212,31 @@ def load(self):
...
         """
-        # currently (pending a new release) astroquery has a race
-        # condition at import time, so putting here rather than at the
-        # module level so that multiple test runners don't run the race
-        from astroquery.simbad import Simbad
-        patchSimbadURL(Simbad)
-        Simbad.add_votable_fields('flux(U)', 'flux(B)', 'flux(V)', 'flux(R)', 'flux(I)', 'flux(J)', 'sptype',
-                                  'parallax', 'pm', 'z_value')
+        # explicitly make a class instance here because:
+        # when using from astroquery.simbad import Simbad and then using
```
Comment (erykoff) [inline spectractor/extractor/targets.py:216]: Should probably quote `from astroquery.simbad import Simbad`

**Review by jeremyneveu — COMMENTED** [review]
Thanks a lot for writing back the tests ! Two things are still missing according to me : the run of the tests/run_full_chain.py and the coverage.

**Review by jeremyneveu — COMMENTED** [review]
This kind of lines are on the top of some tests files : 
```
import matplotlib as mpl  # must be run first! But therefore requires noqa E02 on all other imports
mpl.use('Agg')
```

Then on my laptop I have to comment them to see the plots. Is this mandatory ?

**Pull request merged**